pyccl/__init__.py

```diff
@@ -16,6 +16,14 @@
 # SWIG-generated
 from . import ccllib as lib
 
+# monkey patch for isitgr and fast-pt if Numpy>=1.24
+from packaging.version import parse
+import numpy as np
+if (parse(np.__version__) >= parse('1.24')):
```

Comment:
If it's no big deal, remove the enclosing brackets to keep formatting consistency throughout CCL. I'm approving either way.